cli/cmd/tapRunner.go

```diff
@@ -53,34 +53,31 @@ func RunMizuTap() {
 	defer cancel() // cancel will be called when this function exits
 
 	targetNamespace := getNamespace(kubernetesProvider)
-	if err := updateCurrentlyTappedPods(kubernetesProvider, ctx, targetNamespace); err != nil {
-		mizu.Log.Infof("Error listing pods: %v", err)
-		return
-	}
-
-	if mizu.Config.Tap.DryRun {
-		return
-	}
-
-	urlReadyChan := make(chan string)
-	go func() {
-		mizu.Log.Infof("Mizu is available at http://%s", <-urlReadyChan)
-	}()
-
 	var namespacesStr string
 	if targetNamespace != mizu.K8sAllNamespaces {
 		namespacesStr = fmt.Sprintf("namespace \"%s\"", targetNamespace)
 	} else {
 		namespacesStr = "all namespaces"
 	}
+	go mizu.CheckNewerVersion()
```

Comment:
Is. it ok that it is running now in another routine? (will it write the message of new version in the middle?)

Reply:
good catch